quicklogic/common/toolchain_wrappers/symbiflow_generate_constraints

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+set -e
+
+MYPATH=`realpath $0`
+MYPATH=`dirname ${MYPATH}`
+
+PCF=$1
+EBLIF=$2
+NET=$3
+PART=$4
+DEVICE=$5
+ARCH_DEF=$6
+
+if [[ "$DEVICE" =~ ^(qlf_k4n8_qlf_k4n8)$ ]];then
+	PINMAPCSV="qlf_k4n8-qlf_k4n8_umc22_24x24.csv"
+	DEVICE_1="qlf_k4n8-qlf_k4n8_umc22"
+	PINMAPXML="interface-mapping_24x24.xml"
+else
```

Comment:
can there be `else`? I suppose we should fail here

Reply:
I've rewritten the condition so that it falls back to generic file names for other devices than `qlf_k4n8`. The actual check if the device provided by the user is supported is performed in `ql_symbiflow` so no need for failing here.